src/kontrol/cli.py

```diff
@@ -315,6 +315,13 @@ def parse(s: str) -> list[T]:
         action='store_true',
         help="Do not call 'forge build' during kompilation.",
     )
+    build.add_argument(
+        '--no-silence-warnings',
+        dest='no_silence_warnings',
+        default=None,
+        action='store_true',
+        help='Do not silence K compiler warnings',
```

Comment:
```suggestion
        help='Do not silence K compiler warnings.',
```
I know we're not very consistent about it, but the rest of `build` help messages seem to end with dots :)

Reply:
Thanks for the catch!!